### PR TITLE
Update tdm and blitz scoreboard

### PR DESCRIPTION
**Pull request by Pablete1234**
What thomblr did:
  -  Display TDM and Blitz scoreboard using "number + team" 0 Red Team
  -  [Displays TDM and Blitz ontop of other objectives (ctw/dtc/something)](http://i.imgur.com/PcuuaQM.png)

What I added:
  -  [Sorts blitz and tdm scoreboards, the team with the most score goes first](https://i.gyazo.com/331df7c679a77287d57ee82f982c9c86.png)
  -  TDM max used to be --- MAX ---  250, now it's 10/250 Blue Team
  -  [Add a blank name between other objectives and TDM/Blitz](http://i.imgur.com/BkMd4bT.png)
  -  Fixed scoreboard titles

Scoreboard titles now:
  -  There are 4 objectives, Wools, Cores, Monuments, and Hills
  -  There is Score , or Blitz (Blitz, Blitz: Rage, or Custom name)
  -  If there is only 1 type of objective, the name is used
  -  If there are more than 1 diferent type of objectives, "Objectives" is used
  -  If the match only has TDM or TDM + hills, "Score" is used, if it has any other objective, "Objectives" is used
  -  If there is a Blitz custom name it will prioritize it always
  -  If Blitz is the only gamemode it uses "Blitz, Blitz: Rage. the custom set one". if it has other objectives, the scoreboard will be called "Remaining Players" (it's what OCN uses)

### CODE REVIEW

**Comment by twizmwazin** [comment]
to rebase, just use `git rebase <branch>`. On my computer, my repo is called origin. So I can just do `git rebase origin/master`.

File: src/main/java/in/twizmwaz/cardinal/module/modules/scoreboard/ScoreboardModule.java [hunk 1]
```diff
@@ -477,38 +456,77 @@ public void renderTeamTitle(TeamModule teamModule) {
...
+                int int2 = Integer.parseInt(split2[0]);
+                return int1 - int2;
+                // -1 = str1 < str2
+                // 0 = str1 = str2
+                // 1 = str1 > str2
+            }
+        });
+        for (int i = 0; i < fullNames.size(); i++) {
+            Bukkit.getServer().getConsoleSender().sendMessage(i + ": " + fullNames.get(i));
+        }
```
Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/module/modules/scoreboard/ScoreboardModule.java]: debug code, i'll remove that

**Comment by Pablete1234** [comment]
@iEli2tyree011 opinions?

**Comment by TiTo-418** [comment]
<a href="http://imgur.com/BaFOw0X"></a>

**Comment by Pablete1234** [comment]
@TiTo-418 not a result of this PR

**Comment by Pablete1234** [comment]
@twizmwazin @iEli2tyree011 i'd say this is finnished, is there anything else I should do? also, I want to ask if I both "renderTeamScore()" and "renderTeamBlitz()" should be able to get updated by themselves, or should be always via the "update()", because if that's the case, i could remove "minBlitzScore" and "BlitzScore" and use "update()" in player change team event instead

**Comment by Pablete1234** [comment]
this has been tested for a couple weeks in some servers, and it gave no problems, i'm still waiting for @iEli2tyree011 's feedback...

**Comment by Pablete1234** [comment]
@TiTo-418 fixed your issue with different type of objectives (wools/cores/mon/hills & tdm/blitz) in the same map
How it works atm:
  -  There are 4 objectives, Wools, Cores, Monuments, and Hills
  -  There is Score , or Blitz (Blitz, Blitz: Rage, or Custom name)
  -  If there is only 1 type of objective, the name is used
  -  If there are more than 1 diferent type of objectives, "Objectives" is used
  -  If the match only has TDM or TDM + hills, "Score" is used, if it has any other objective, "Objectives" is used
  -  Finally, Blitz alone uses "Blitz, Blitz: Rage. or a custom set one". if it has other objectives, the scoreboard will be called "Remaining Players" (it's what OCN uses)
Opinions? anything  I should do different?

**Comment by Eudaldca** [comment]
Tested with a bunch of maps, it works beautifully :)

**Comment by thomblr** [comment]
Tested too and working as needed :+1:

**Comment by Prodigy4532** [comment]
Tested with 12 TDM maps and works fine, Scoreboard KotH maps =

**Comment by Pablete1234** [comment]
I know of a small bug, in the display name, if it has a blitz custom name + other gamemodes, it still says players remaining, it should prioritize the custom name, i'll fix it soon

**Comment by Pablete1234** [comment]
@Eudaldca @thomblr @Prodigy4532 could you do a last test?

**Comment by Alfalik** [comment]
This works as expected, no bugs/errors found.

**Comment by Pablete1234** [comment]
@iEli2tyree011 do you think anything else should be done here? if not, merge and i can rebase the "Update scoreboard" PR and start working from there

**Comment by iEli2tyree011** [comment]
@Pablete1234 Since it appears to be tested with no problems, I'll merge it

**Pull request merged**